zerver/openapi/zulip.yaml

```diff
@@ -14343,6 +14350,13 @@ paths:
                           Present if `realm` is present in `fetch_event_types`.
 
                           The name of the organization, used in login pages etc.
+                      realm_require_unique_names:
+                        type: boolean
+                        description: |
+                          Indicates whether the organization enforces users to have
+                          same full_names (if set to true) or not.
+
+                          **Changes**: New in Zulip 9.0 (feature level 244).
```

Comment:
It's always good to be specific about what happened before that feature level. Add 

> Previously, the Zulip server could not be configured to enforce unique names.